my-dsl.md

```diff
@@ -2,37 +2,78 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I looked at the language [Cg][1].
+
+[1]: https://developer.nvidia.com/cg-toolkit
+
 
 # Domain
 _Describe the language's domain in five words._
 
+vertex/pixel shading for GPUs
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+According to the [Cg tutorial][2], section 1.4.2, Cg is executed to interact
+with the OpenGL or Direct3D API for your GPU. The way this works is for a Cg
+program to be translated into OpenGL or Direct3D, then for the translated
```

Comment:
I agree. It might also be interesting to know whether the translation does anything special for different GPUs or GPU programming languages. Is it a perfect mapping from Cg to those languages / architectures?